src/main/resources/assets/draconicevolution/lang/pt_BR.lang

```diff
@@ -50,21 +47,21 @@ tile.draconicevolution:energy_crystal.wireless.draconic.name=Cristal Draconiano
 tile.draconicevolution:fusion_crafting_core.name=Núcleo de Fabricação por Fusão
 tile.draconicevolution:crafting_injector_basic.name=Injetor para Fabricação por Fusão
 tile.draconicevolution:crafting_injector_wyvern.name=Injetor Wyvern para Fabricação por Fusão
-tile.draconicevolution:crafting_injector_draconic.name=Injeteor Draconiano para Fabricação por Fusão
+tile.draconicevolution:crafting_injector_draconic.name=Injeteor Draconiano para Fabricação por Fusão 
 tile.draconicevolution:crafting_injector_chaotic.name=Injetor Caótico para Fabricação por Fusão
 
 tile.draconicevolution:creative_rf_source.name=Fonte de RF Criativa
 
 
 //Item Information ----------------------------------------------------------------------------------------------------------------------------------------
-item.draconicevolution:awakened_core.name=Núcleo Evoluído
+item.draconicevolution:awakened_core.name=Núcleo Draconiano
 item.draconicevolution:draconium_ingot.name=Lingote de Dracônio
 item.draconicevolution:draconium_dust.name=Pó de Dracônio
 item.draconicevolution:draconium_blend.name=Mistura de Dracônio (Depreciada)
 item.draconicevolution:dragon_heart.name=Coração de Dragão
 item.draconicevolution:wyvern_core.name=Núcleo Wyvern
 item.draconicevolution:sun_focus.name=Foco Solar
-item.draconicevolution:draconic_core.name=Núcleo Draconiano
+item.draconicevolution:draconic_core.name=Núcleo Evoluído
```

Comment:
Isso não faz sentido nenhum, você está mudando o nome do núcleo Draconiano para Evoluído. A intenção é traduzir, não criar um nome novo.

Reply:
Sim. A ideia é criar uma sequência lógica sem usar nomes feios:
Primeiro Evoluído - por ser apenas diamante que evoluiu devido ao uso de lingotes de dracônio. (Assim como a gente vê no lore dos equipamentos e da armadura Wyvern)
Depois Wyvern
Depois Draconiano - Que é o nível do Awakened Draconium.
E então Caótico.

E isso, por dois motivos:
Primeiro: Awakened significa despertado, ou acordado. E esse é um péssimo nome para colocar em português.
Segundo: Se vc for reparar, verá que o núcleo chamado 'Draconiano' é de nível 1. Ou seja, anterior ao Wyvern. Porém a armadura que é feita com dracônio despertado (Nome feio), mesmo sendo de nível 3, se chama armadura **draconiana**, o que dá a ideia de que é feita com um núcleo draconiano (Em português), o que não é verdade. Por isso, ao invés de colocar "draconiano e draconiano despertado", coloquei assim.
Só jogar usando a fabricação por fusão que vc verá que fica confuso: Quando a fabricação por fusão mostra o nível necessário para aquela fusão, vc não sabe se está falando do nível 1 ou do 3. Por isso fis essa diferenciação.

Não fiz nada que realmente comprometesse o lore, nem a ideia principal que está implícita por trás. É apenas uma diferenciação para evitar confusão.

Reply:
Awakened ou Acordado, literalmente o mesmo sentido em inglês e português, em traduções não se altera o conteúdo, ainda mais pra quem vê ou lê coisas em ambas línguas, vai ficar incompatível.

Reply:
Do que você está falando?